### PR TITLE
Add an issue template for feature requests

### PR DESCRIPTION
**Pull request by j-f1**
Let me know if you have any feedback on the language in the template. The fields are based on (i.e. shamelessly copied and pasted from) the [Babel](https://github.com/babel/babel/issues/new?assignees=&labels=i%3A+needs+triage%2Ci%3A+enhancement&template=feature_request.yml), [GitHub Desktop](https://github.com/desktop/desktop/issues/new?assignees=&labels=&template=feature_request.yaml), and [Mastodon](https://github.com/mastodon/mastodon/issues/new?assignees=&labels=suggestion&template=2.feature_request.yml) feature request forms.

### CODE REVIEW

**Review by MatissJanis — APPROVED** [review]
This is great! I'm happy to merge this but cc @rich-howell for another review.

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+name: Feature request
+description: Request a missing feature
+title: '[Feature] '
+labels: ['feature', 'needs triage']
+body:
+  - type: markdown
+    id: intro-md
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this feature request! Please ensure you provide as much information as possible so we can better understand what you want us to add and so we can come up with the best solution for everyone.
```
Comment (rich-howell) [inline .github/ISSUE_TEMPLATE/feature-request.yml:10]: Maybe change this to "what your posing to be added and so we can come up with the best solution for everyone." so it takes away the expectation that what is asked will get deployed

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    attributes:
+      label: '💻'
+      description: (Optional) Please check this box if you’re willing to open a PR to implement this feature. We’ll help you get started and answer any questions you have along the way :)
+      options:
+        - label: Would you like to implement this feature?
+  - type: checkboxes
+    id: existing-issue
+    attributes:
+      label: 'Verified feature request does not already exist?'
+      description: 'Please search to see if an issue already exists for the feature you have requested.'
```
Comment (rich-howell) [inline .github/ISSUE_TEMPLATE/feature-request.yml:21]: Should we include PR's sometimes people just submit PR's without an issue.

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      options:
+        - label: 'I have searched and found no existing issue'
+          required: true
+    validations:
+      required: true
+  - type: textarea
+    id: pitch
+    attributes:
+      label: 'Pitch: what problem are you trying to solve?'
+      description: Please concisely describe the problem that motivated your feature request.
```
Comment (rich-howell) [inline .github/ISSUE_TEMPLATE/feature-request.yml:31]: Please describe in as much detail, the problem that motivated you to submit a feature request.

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    validations:
+      required: true
+  - type: textarea
+    id: solution
+    attributes:
+      label: Describe the solution you’d like.
```
Comment (rich-howell) [inline .github/ISSUE_TEMPLATE/feature-request.yml:37]: Describe your ideal solution that would solve this problem.

**Review by rich-howell — CHANGES_REQUESTED** [review]
Hey @j-f1 

Submitted some comments in feature-request.yml - what do you think?

Cheers
Rich

**Comment by j-f1** [comment]
Took those into account @rich-howell, lmk what you think about the updates!

**Review by rich-howell — APPROVED** [review]
Approved. Thanks @j-f1

**Pull request merged**